src/plonk/verifier.rs

```diff
@@ -261,12 +264,21 @@ impl<C: CurveAffine> Proof<C> {
         }
 
         // Verify the opening proof
-        self.opening.verify(
-            params,
-            &mut transcript,
-            x_6,
-            &f_commitment.to_affine(),
-            f_eval,
-        )
+        let default_msm = MSM::default(&params);
```

Comment:
I think we should have a method called `msm` on `Params` instead of doing `MSM::default()`, but in any case, we shouldn't call it `default()` because that conflicts with the `Default::default()` method signature, which is in the prelude and so in scope.